app/jobs/bulkrax/import_work_collection_job.rb

```diff
@@ -19,5 +20,22 @@ def perform(*args)
       end
     end
     # rubocop:enable Rails/SkipsModelValidations
+
+    def add_user_to_permission_template!(entry)
+      user                = ::User.find(entry.importerexporter.user_id)
+      collection          = entry.factory.find
+      permission_template = Hyrax::PermissionTemplate.find_by(source_id: collection.id)
+
+      if permission_template.present?
+        Hyrax::PermissionTemplateAccess.create!(
+          permission_template_id: permission_template.id,
+          agent_id: user.email,
+          agent_type: 'user',
+          access: 'manage'
+        )
```

Comment:
Need to call `collection.reset_access_controls!`, see [hyrax/spec/abilities/collection_ability_spec.rb](https://github.com/samvera/hyrax/blob/master/spec/abilities/collection_ability_spec.rb#L42-L47)